deepseg.cc

```diff
@@ -257,7 +301,7 @@ void calc_mask(calcinfo_t &info, timinginfo_t &ti) {
 	uint8_t* out = (uint8_t*)info.ofinal.data;
 
 	// find class with maximum probability
-	if (strstr(info.modelname,"deeplab")) {
+	if (modeltype::DeepLab == info.modeltype) {
```

Comment:
Probably better of as a switch/case, thus ensuring to cover all cases. Currently missing the `modeltype::Unknown`/`default` case.

Reply:
Changed to switch / case instead. 

Quit early if model is unknown since then we do not know how to parse the output.